lang/interpolate.go

```diff
@@ -254,3 +323,42 @@ func concatExprListIntoCall(exprs []interfaces.Expr) (interfaces.Expr, error) {
 		},
 	}, nil
 }
+
+// simplifyExprList takes a list of *ExprStr and *ExprVar and groups the
+// sequential *ExprStr's together. If you pass it a list of Expr's that contains
+// a different type of Expr, then this will error.
+func simplifyExprList(exprs []interfaces.Expr) ([]interfaces.Expr, error) {
+	last := false
+	result := []interfaces.Expr{}
+
+	for _, x := range exprs {
+		switch v := x.(type) {
+		case *ExprStr:
+			if !last {
+				last = true
+				result = append(result, x)
+				continue
+			}
+
+			// combine!
+			last := result[len(result)-1] // there has to be at least one
```

Comment:
This `last` here confused me because it shadows the `var last bool` above, but this is an `Expr`. Maybe pick a different name so it's clearly a different type?

Reply:
Yeah! Good catch-- I don't know who wrote that nonsense!